### PR TITLE
Add the_longest_night encounter set 

### PR DESCRIPTION
**Pull request by fspoettel**

### CODE REVIEW

File: pack/fhv/fhvc.json [hunk 1]
```diff
@@ -146,15 +146,296 @@
...
+    "pack_code": "fhvc",
+    "position": 130,
+    "quantity": 1,
+    "shroud": 4,
+    "text": "The Farmhouse cannot have player card attachments, traps, or decoys.\n[free] During your turn, spend 1 [per_investigator] resources: <b>Move</b> <i>(to a connecting location).</i>\n[action]: Gain 1 clue<i> (from the token pool)</i>. (Group limit once per round.)",
+    "traits": "Sanctum.",
+    "type_code": "location"
+  },
+  {
+    "back_flavor": "The old farm is riddled with scraps, desiccated livestock, rotten feed, and rusted farming equipment.",
```
Comment (fspoettel) [inline pack/fhv/fhvc.json:223]: There is a typo in the original card text (_dessicated_ vs _desiccated_), I took the liberty to fix it.

File: pack/fhv/fhvc.json [hunk 2]
```diff
@@ -165,10 +446,266 @@
...
+    "pack_code": "fhvc",
+    "position": 143,
+    "quantity": 1,
+    "subname": "Starving Abomination",
+    "text": "Hunter. Massive. Retaliate.\n<b>Prey</b> - Lead investigator.\nUrsine Hybrid gets +3 [per_investigator] health and ignores barriers and decoys.\nWhile this enemy is moving, each [[Field]] location is considered connected to one another.",
+    "traits": "Creature. Monster. Mutated. Elite.",
+    "type_code": "enemy",
+    "victory": 2
+  },
+  {
```
Comment (fspoettel) [inline pack/fhv/fhvc.json:474]: Some of the enemies in this set have variants with different spawn instructions.

I used ids `{code}a`, `{code}b`, `{code}c` to disambiguate the different versions.

**Comment by fspoettel** [comment]
ping @drawntotheflame.

File: pack/fhv/fhvc.json [hunk 3]
```diff
@@ -165,10 +446,266 @@
...
+    "encounter_position": 29,
+    "enemy_damage": 2,
+    "enemy_evade": 2,
+    "enemy_fight": 2,
+    "faction_code": "mythos",
+    "health": 3,
+    "illustrator": "Peter Polach",
+    "name": "Equine Hybrid",
+    "pack_code": "fhvc",
+    "position": 147,
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json:638]: I wonder if for b and c we need to boost the position by 1 (so 148 and 149). I think one for @zzorba to test when he has this live (or you can, @fspoettel ?) as I think having multiple cards with the same position might make the cards display somewhat oddly? Not 100% sure on that though...

File: pack/fhv/fhvc.json [hunk 4]
```diff
@@ -165,10 +446,266 @@
...
+  {
+    "code": "10646",
+    "encounter_code": "the_longest_night",
+    "encounter_position": 25,
+    "enemy_damage": 1,
+    "enemy_evade": 3,
+    "enemy_fight": 3,
+    "enemy_horror": 2,
+    "faction_code": "mythos",
+    "flavor": "I suppose that, like any goat, it eats everything.\n<cite>Dr. Rosa Marquez, Miskatonic Survey, 1926</cite>",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json]:
```suggestion
    "flavor": "I suppose that, like any goat, it eats everything.\n - Dr. Rosa Marquez, <u>Miskatonic Survey, 1926</u>",
```

Comment (drawntotheflame) [reply pack/fhv/fhvc.json]: Generally speaking, we've erred towards keeping it simple and just using a dash and underline for titles. So I'd prefer for this for consistency across the db.

File: pack/fhv/fhvc.json [hunk 5]
```diff
@@ -165,10 +446,266 @@
...
+  {
+    "code": "10645a",
+    "encounter_code": "the_longest_night",
+    "encounter_position": 22,
+    "enemy_damage": 1,
+    "enemy_evade": 1,
+    "enemy_fight": 3,
+    "enemy_horror": 1,
+    "faction_code": "mythos",
+    "flavor": "…strangely, they didn't howl. They laughed.\n<cite>Dr. Rosa Marquez, Miskatonic Survey, 1926</cite>",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json]:
```suggestion
    "flavor": "...strangely, they didn't howl. They laughed.\n - Dr. Rosa Marquez, <u>Miskatonic Survey, 1926</u>",
```

File: pack/fhv/fhvc.json [hunk 6]
```diff
@@ -165,10 +446,266 @@
...
+  {
+    "code": "10645b",
+    "encounter_code": "the_longest_night",
+    "encounter_position": 23,
+    "enemy_damage": 1,
+    "enemy_evade": 1,
+    "enemy_fight": 3,
+    "enemy_horror": 1,
+    "faction_code": "mythos",
+    "flavor": "…strangely, they didn't howl. They laughed.\n<cite>Dr. Rosa Marquez, Miskatonic Survey, 1926</cite>",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json]:
```suggestion
    "flavor": "...strangely, they didn't howl. They laughed.\n - Dr. Rosa Marquez, <u>Miskatonic Survey, 1926</u>",
```

File: pack/fhv/fhvc.json [hunk 7]
```diff
@@ -165,10 +446,266 @@
...
+  {
+    "code": "10645c",
+    "encounter_code": "the_longest_night",
+    "encounter_position": 24,
+    "enemy_damage": 1,
+    "enemy_evade": 1,
+    "enemy_fight": 3,
+    "enemy_horror": 1,
+    "faction_code": "mythos",
+    "flavor": "…strangely, they didn't howl. They laughed.\n<cite>Dr. Rosa Marquez, Miskatonic Survey, 1926</cite>",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json]:
```suggestion
    "flavor": "...strangely, they didn't howl. They laughed.\n - Dr. Rosa Marquez, <u>Miskatonic Survey, 1926</u>",
```

File: pack/fhv/fhvc.json (hunk 2 above)
Comment (drawntotheflame) [reply pack/fhv/fhvc.json:474]: I think this is a good idea. I've commented below that maybe the position for B and C needs to be changed - something to check in terms of how the cards display on the card list once on the DB.

File: pack/fhv/fhvc.json [hunk 8]
```diff
@@ -146,15 +146,296 @@
...
+    "encounter_position": 16,
+    "faction_code": "neutral",
+    "health": 10,
+    "illustrator": "Peter Polach",
+    "is_unique": true,
+    "name": "The Captives",
+    "pack_code": "fhvc",
+    "position": 141,
+    "quantity": 1,
+    "sanity": null,
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json]: You can cut this line - not necessary when it's a null value for health/sanity.

File: pack/fhv/fhvc.json [hunk 9]
```diff
@@ -146,15 +146,296 @@
...
+    "pack_code": "fhvc",
+    "position": 135,
+    "quantity": 1,
+    "shroud": 3,
+    "text": "[free] During your turn, if Pasture has a trap on it: <b>Move</b> <i>(to a connecting location)</i>.\n<b>Forced</b> – After a ready, unengaged enemy moves into Pasture via a keyword, if there are no barriers at Pasture: Resolve that keyword again.",
+    "traits": "Farm.",
+    "type_code": "location"
+  },
+  {
+    "back_flavor": "All the verdure was going grey, and was developing a highly singular quality of brittleness.\n<cite>H.P. Lovecraft, \"The Colour Out of Space\"</cite>",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json]:
```suggestion
    "back_flavor": "All the verdure was going grey, and was developing a highly singular quality of brittleness.\n - H.P. Lovecraft, \"The Colour Out of Space\"",
```

Comment (drawntotheflame) [reply pack/fhv/fhvc.json]: NB we've not used `<cite>` elsewhere on the DB, so please don't introduce it now - let's stick with keeping the flavor style as consistent as we can (NB2, I know that the style of Lovecraft's own name has changed across cards - H.P. Lovecraft, HP Lovecraft, H. P. Lovecraft, etc - one day I'll work through making them all the same, even if FFG can't manage that!)

File: pack/fhv/fhvc.json [hunk 10]
```diff
@@ -146,15 +146,296 @@
...
+    "encounter_code": "the_longest_night",
+    "encounter_position": 11,
+    "faction_code": "mythos",
+    "illustrator": "Nele Diel",
+    "name": "Outer Fields",
+    "pack_code": "fhvc",
+    "position": 136,
+    "quantity": 1,
+    "shroud": 2,
+    "subname": "Bloodied Path",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json]:
```suggestion
    "subname": "Bloodied Paths",
```

File: pack/fhv/fhvc.json [hunk 11]
```diff
@@ -146,15 +146,296 @@
...
+    "text": "[action]: Move an enemy to a connecting location. (Group limit once per game.)\n[reaction] After you discover the last clue from this location: Heal 2 damage from The Captives. (Group limit once per game.)",
+    "traits": "Field. Blight.",
+    "type_code": "location"
+  },
+  {
+    "back_flavor": "All the verdure was going grey, and was developing a highly singular quality of brittleness.\n<cite>H.P. Lovecraft, \"The Colour Out of Space\"</cite>",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json]:
```suggestion
    "back_flavor": "All the verdure was going grey, and was developing a highly singular quality of brittleness.\n - H.P. Lovecraft, \"The Colour Out of Space\"",
```

File: pack/fhv/fhvc.json [hunk 12]
```diff
@@ -146,15 +146,296 @@
...
+    "encounter_position": 12,
+    "faction_code": "mythos",
+    "illustrator": "Nele Diel",
+    "name": "Outer Fields",
+    "pack_code": "fhvc",
+    "position": 137,
+    "quantity": 1,
+    "shroud": 2,
+    "subname": "Desolate Hills",
+    "text": "[free] During your turn: Look at the top 1 [per_investigator] cards of the Enemy deck. Discard any number of them and put the rest back on top in any order. (Group limit once per game.)",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json]: We prefer `[fast]` to `[free]` on the DB, please.

File: pack/fhv/fhvc.json [hunk 13]
```diff
@@ -146,15 +146,296 @@
...
+    "traits": "Field. Blight.",
+    "type_code": "location"
+  },
+  {
+    "back_flavor": "All the verdure was going grey, and was developing a highly singular quality of brittleness.\n<cite>H.P. Lovecraft, \"The Colour Out of Space\"</cite>",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json]:
```suggestion
    "back_flavor": "All the verdure was going grey, and was developing a highly singular quality of brittleness.\n - H.P. Lovecraft, \"The Colour Out of Space\"",
```

File: pack/fhv/fhvc.json [hunk 14]
```diff
@@ -146,15 +146,296 @@
...
+    "position": 138,
+    "quantity": 1,
+    "shroud": 3,
+    "subname": "Blighted Cornfields",
+    "text": "[reaction] After you discover the last clue from this location: Place 1 barrier at this location.\n[free] During your turn: Choose a barrier at any location and move it to any other location. (Limit once per turn.)",
+    "traits": "Field. Blight.",
+    "type_code": "location"
+  },
+  {
+    "back_flavor": "All the verdure was going grey, and was developing a highly singular quality of brittleness.\n<cite>H.P. Lovecraft, \"The Colour Out of Space\"</cite>",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json]:
```suggestion
    "back_flavor": "All the verdure was going grey, and was developing a highly singular quality of brittleness.\n - H.P. Lovecraft, \"The Colour Out of Space\"",
```

File: pack/fhv/fhvc.json [hunk 15]
```diff
@@ -146,15 +146,296 @@
...
+    "position": 139,
+    "quantity": 1,
+    "shroud": 3,
+    "subname": "Scorched Knoll",
+    "text": "[reaction] After you discover the last clue from this location: Place 1 trap at this location.\n[free] During your turn: Choose a trap at any location and move it to any other location. (Limit once per turn.)",
+    "traits": "Field. Blight.",
+    "type_code": "location"
+  },
+  {
+    "back_flavor": "All the verdure was going grey, and was developing a highly singular quality of brittleness.\n<cite>H.P. Lovecraft, \"The Colour Out of Space\"</cite>",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json]:
```suggestion
    "back_flavor": "All the verdure was going grey, and was developing a highly singular quality of brittleness.\n - H.P. Lovecraft, \"The Colour Out of Space\"",
```

File: pack/fhv/fhvc.json (hunk 1 above)
Comment (drawntotheflame) [reply pack/fhv/fhvc.json:223]: I agree 100% with this fix.

File: pack/fhv/fhvc.json [hunk 16]
```diff
@@ -146,15 +146,296 @@
...
+    "encounter_code": "the_longest_night",
+    "encounter_position": 8,
+    "faction_code": "mythos",
+    "illustrator": "Pixoloid Studios",
+    "name": "Coop",
+    "pack_code": "fhvc",
+    "position": 133,
+    "quantity": 1,
+    "shroud": 4,
+    "text": "While there is a decoy on Coop, this location gets -2 shroud.\nEach enemy at Coop gets +1 fight and + 1 evade.\n[action]: Draw 1 set-aside Fire! treachery. Shuffle 1 copy of a set-aside Fire! treachery into the encounter deck. (Limit once per round.)",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json]:
```suggestion
    "text": "While there is a decoy on Coop, this location gets -2 shroud.\nEach enemy at Coop gets +1 fight and +1 evade.\n[action]: Draw 1 set-aside Fire! treachery. Shuffle 1 copy of a set-aside Fire! treachery into the encounter deck. (Limit once per round.)",
```

File: pack/fhv/fhvc.json [hunk 17]
```diff
@@ -146,15 +146,296 @@
...
+    "encounter_position": 9,
+    "faction_code": "mythos",
+    "flavor": "The barn looks like it could collapse at any moment.",
+    "illustrator": "Pixoloid Studios",
+    "name": "Barn",
+    "pack_code": "fhvc",
+    "position": 134,
+    "quantity": 1,
+    "shroud": 3,
+    "text": "Enemies at Barn get +1 damage value.\nWhen Barn is revealed: Place the set-aside Ajax story asset at this location <b>([codex] Ω)</b>.",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json]:
```suggestion
    "text": "Enemies at Barn get +1 damage value.\n[reaction] When Barn is revealed: Place the set-aside Ajax story asset at this location <b>([codex] Ω)</b>.",
```

File: pack/fhv/fhvc.json [hunk 18]
```diff
@@ -146,15 +146,296 @@
...
+    "name": "The Longest Night",
+    "pack_code": "fhvc",
+    "position": 129,
+    "quantity": 1,
+    "stage": 1,
+    "text": "[free] During your turn, spend 1 clue: Place 1 barrier, decoy, or trap at your location.\n[action]: <b>Resign.</b> Then, if each undefeated investigator has resigned, <b>(→R3)</b> <i>(page 50).</i>\n<b>Objective</b> – Survive the night and protect as many captives as you can. <i>(Do not advance until you are instructed.)</i>",
+    "type_code": "act"
+  },
+  {
+    "back_flavor": "The aspect of the whole farm was shocking— greyish withered grass and leaves on the ground, vines falling in brittle wreckage from archaic walls and gables.\n<cite>H.P. Lovecraft, \"The Colour Out of Space\"</cite>",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json]:
```suggestion
    "back_flavor": "The aspect of the whole farm was shocking - greyish withered grass and leaves on the ground, vines falling in brittle wreckage from archaic walls and gables.\n - H.P. Lovecraft, \"The Colour Out of Space\"",
```

Comment (drawntotheflame) [reply pack/fhv/fhvc.json]: NB we don't use em dashes on the db, just a simple hyphen is fine.

File: pack/fhv/fhvc.json [hunk 19]
```diff
@@ -146,15 +146,296 @@
...
+    "encounter_code": "the_longest_night",
+    "encounter_position": 5,
+    "faction_code": "mythos",
+    "illustrator": "Pixoloid Studios",
+    "name": "The Farmhouse",
+    "pack_code": "fhvc",
+    "position": 130,
+    "quantity": 1,
+    "shroud": 4,
+    "text": "The Farmhouse cannot have player card attachments, traps, or decoys.\n[free] During your turn, spend 1 [per_investigator] resources: <b>Move</b> <i>(to a connecting location).</i>\n[action]: Gain 1 clue<i> (from the token pool)</i>. (Group limit once per round.)",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json]:
```suggestion
    "text": "The Farmhouse cannot have player card attachments, traps, or decoys.\n[free] During your turn, spend 1 [per_investigator] resources: <b>Move</b> <i>(to a connecting location)</i>.\n[action]: Gain 1 clue <i>(from the token pool)</i>. (Group limit once per round.)",
```

File: pack/fhv/fhvc.json [hunk 20]
```diff
@@ -146,15 +146,296 @@
...
+    "encounter_position": 4,
+    "faction_code": "mythos",
+    "flavor": "You must protect the captives!",
+    "illustrator": "Mauro Dal Bo",
+    "name": "The Longest Night",
+    "pack_code": "fhvc",
+    "position": 129,
+    "quantity": 1,
+    "stage": 1,
+    "text": "[free] During your turn, spend 1 clue: Place 1 barrier, decoy, or trap at your location.\n[action]: <b>Resign.</b> Then, if each undefeated investigator has resigned, <b>(→R3)</b> <i>(page 50).</i>\n<b>Objective</b> – Survive the night and protect as many captives as you can. <i>(Do not advance until you are instructed.)</i>",
```
Comment (drawntotheflame) [inline pack/fhv/fhvc.json]:
```suggestion
    "text": "[free] During your turn, spend 1 clue: Place 1 barrier, decoy, or trap at your location.\n[action]: <b>Resign.</b> Then, if each undefeated investigator has resigned, <b>(→R3)</b> <i>(page 50)</i>.\n<b>Objective</b> – Survive the night and protect as many captives as you can. <i>(Do not advance until you are instructed.)</i>",
```

**Review by drawntotheflame — COMMENTED** [review]
Great job @fspoettel - a few little tweaks / style things from previous ways we've done things on the DB.

Worth noting, I work from the end to the start, so if you see comments late on, that's why!

File: pack/fhv/fhvc.json (hunk 3 above)
Comment (fspoettel) [reply pack/fhv/fhvc.json:638]: I already tested this on arkhamdb locally and it is fine.

File: pack/fhv/fhvc.json (hunk 18 above)
Comment (fspoettel) [reply pack/fhv/fhvc.json]: That's not exactly true, we are using em dashes a bunch in this very file and i.e. eote and tsk.

Comment (drawntotheflame) [reply pack/fhv/fhvc.json]: > That's not exactly true, we are using em dashes a bunch in this very file and i.e. eote and tsk.

ok, please stick to dashes and I'll remove the em dashes on those two packs.

File: pack/fhv/fhvc.json (hunk 3 above)
Comment (drawntotheflame) [reply pack/fhv/fhvc.json:638]: Great, that's good to know.

File: pack/fhv/fhvc.json (hunk 12 above)
Comment (fspoettel) [reply pack/fhv/fhvc.json]: Will do - this also needs a cleanup then, it's used a lot on other packs.

File: pack/fhv/fhvc.json (hunk 3 above)
Comment (fspoettel) [reply pack/fhv/fhvc.json:638]: @drawntotheflame here is what it looks like:

image

**Comment by fspoettel** [comment]
@drawntotheflame implemented all requested feedback and suggestions a while ago, can you ✅ so we can merge this?

**Review by drawntotheflame — APPROVED** [review]
Good stuff!

**Comment by fspoettel** [comment]
@zzorba can you merge this one? 🙏 lack the permissions to do it. thanks!

**Pull request merged**